bluesky_queueserver/worker.py

```diff
@@ -262,35 +257,17 @@ def _conn_received(self, msg):
             # Stop the loop in main thread
             logger.info("Pausing Run Engine")
             pausing_options = ("deferred", "immediate")
+            # TODO: the question is whether it is possible or should be allowed to pause a plan in
```

Comment:
The state machine on the RE does not allow moving from idle -> paused.

Reply:
I meant the transitional states like 'pausing', 'resuming' etc. The possible issue may be that in the unlikely case the command to pause is issued during any of those states, it will be rejected. So depending on how the rest of the system works, the request has to be either retried or the user has to be informed that the request was rejected. But I think processing of those cases may be left for later when we work on exchange protocols.